src/model/Model.ts

```diff
@@ -116,8 +116,19 @@ export class DIVEModel extends DIVENode {
         const worldPos = this.getWorldPosition(this._positionWorldBuffer);
         const oldWorldPos = worldPos.clone();
 
-        // calculate the bottom center of the bounding box and set it to world posion
-        worldPos.y = (this._boundingBox.max.y - this._boundingBox.min.y) / 2;
+        // compute the bounding box
+        this._mesh?.geometry?.computeBoundingBox();
+        const meshBB = this._mesh?.geometry?.boundingBox;
+        console.log(meshBB);
+        // subtract the bounding box min y axis value from the world position y value
+        if (meshBB && this._mesh) {
```

Comment:
Would it be okay for you if we would write like`if (!meshBB || !this.mesh) return;`? This would prevent the additional tab.

The reason is, that there is no additional functionality executed based on the condition. It is just like "don't do anything if the condition is `false`". I hope you know what I mean :)